src/Infrastructure/Repositories/EmployeeRepository.cs

```diff
@@ -45,8 +45,11 @@ public async Task<List<Employee>> GetEmployeesByCountry(string country)
         return await _db.Employees
             .Include(employee => employee.ProjectExperiences)
             .ThenInclude(entity => entity.ProjectExperienceRoles)
+            .Include(employee => employee.ProjectExperiences)

```

Comment:
Er dette rett måte å gjøre det på - at man må include ProjectExperiences to ganger for å kunne joine inn to tables videre?

Reply:
Jepp.

Reply:
https://stackoverflow.com/questions/30072360/include-several-references-on-the-second-level

Jeg kan formatere det litt bedre dog.